tdms/include/arrays.h

```diff
@@ -421,17 +432,29 @@ class Tensor3D {
   }
 };
 
+/**
+ * @brief Stores the fibre modes in the Fourier plane of the objective lens.
+ *
+ * The "Tilde" indicates that these quantities are in a Fourier plane relative
```

Comment:
Sorry, ninja additional suggestion. Did we try out LaTeX & Doxygen?

$\tilde D$ works?

Reply:
No :(

![image](https://user-images.githubusercontent.com/32364977/235196306-c1cd203e-3ae8-4278-a337-9bf4b044d36e.png)